src/app/shared/utils/constants.ts

```diff
@@ -21,8 +21,8 @@ export class Constants {
 
   public static readonly NOTIFICATION_DECISIONS = [
     'In Progress',
-    'Referred for s.11 consideration',
-    'Not referred for s.11 consideration'
+    'Referred to Minister for Designation Decision',
```

Comment:
Now that this value is being checked against in `add-edit-project-notification.component.html` at line 96, it might be ideal to update to be an object like the `documentTypes` object below. That the key can be referenced and if the value every changes again it only needs updating in here.